plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftConnectorTest.java

```diff
@@ -730,6 +730,35 @@ public void testJoinPushdownWithImplicitCast()
         }
     }
 
+    @Test
+    public void testLpadPushdown()
```

Comment:
can we add a simple test case for `lpad` on projection level as well? something like:

`SELECT lpad(l.c_varchar_50, 8, 'aaa') from table1`;

Reply:
Also the test coverage where length is greater and less than than column.

Reply:
sure, I can add one with greater than the column. We already have something simillar.

assertThat(query(session, "SELECT *, lpad(c_varchar_50, 20, 'aaa') FROM " + leftTable.getName()))
                    .isFullyPushedDown();